src/software/simulated_tests/requirements.txt

```
@@ -1,2 +1,2 @@
 pytest==6.2.5
-
+decorator
```

I think to avoid changing behavior over time, we should set the version of this library (`5.1.1` seems to be the latest)
```suggestion
decorator==5.1.1
```